src/query/insert.rs

```diff
@@ -318,7 +318,14 @@ where
     }
 
     // helper function for do_nothing in Insert<A>
-    pub fn from_insert(insert: Insert<A>) -> Self {
+    pub fn from_insert(mut insert: Insert<A>) -> Self {
+        let primary_keys = <A::Entity as EntityTrait>::PrimaryKey::iter();
+        insert.query.on_conflict(
+            OnConflict::columns(primary_keys.clone())
+                .do_nothing_on(primary_keys)
+                .to_owned(),
+        );
+
```

Comment:
I don't think this code belongs here. `on_empty_do_nothing` does not mean `on_conflict_do_nothing`

Reply:
Updated! https://github.com/SeaQL/sea-orm/pull/2244/commits/27ce67e20942a3ac9885c23590e73596905ab9c6